new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItOperatorTwoDomains.java

```diff
@@ -0,0 +1,690 @@
+                  .putRequestsItem("storage", Quantity.fromString("5Gi"))))
+          .metadata(new V1ObjectMetaBuilder()
+              .withName(pvcName)
+              .withNamespace(domainNamespace)
+              .build()
+              .putLabelsItem("weblogic.resourceVersion", "domain-v2")
+              .putLabelsItem("weblogic.domainUid", domainUid));
+
+      String labelSelector = String.format("weblogic.domainUid in (%s)", domainUid);
+      createPVPVCAndVerify(v1pv, v1pvc, labelSelector, domainNamespace);
+
+      // run create a domain on PV job using WLST
+      runCreateDomainOnPVJobUsingWlst(pvName, pvcName, domainUid, domainNamespace);
+
+      // create the domain custom resource configuration object
+      logger.info("Creating domain custom resource");
+      Domain domain = new Domain()
+          .apiVersion(DOMAIN_API_VERSION)
+          .kind("Domain")
+          .metadata(new V1ObjectMeta()
+              .name(domainUid)
+              .namespace(domainNamespace))
+          .spec(new DomainSpec()
+              .domainUid(domainUid)
+              .domainHome("/shared/domains/" + domainUid)
+              .domainHomeSourceType("PersistentVolume")
+              .image(image)
+              .imagePullSecrets(isUseSecret ? Arrays.asList(
+                  new V1LocalObjectReference()
+                      .name(OCR_SECRET_NAME))
+                  : null)
+              .webLogicCredentialsSecret(new V1SecretReference()
+                  .name(wlSecretName)
+                  .namespace(domainNamespace))
+              .includeServerOutInPodLog(true)
+              .logHomeEnabled(Boolean.TRUE)
+              .logHome("/shared/logs/" + domainUid)
+              .dataHome("")
+              .serverStartPolicy("IF_NEEDED")
+              .serverPod(new ServerPod()
+                  .addEnvItem(new V1EnvVar()
+                      .name("JAVA_OPTIONS")
+                      .value("-Dweblogic.StdoutDebugEnabled=false"))
+                  .addEnvItem(new V1EnvVar()
+                      .name("USER_MEM_ARGS")
+                      .value("-Djava.security.egd=file:/dev/./urandom "))
+                  .addVolumesItem(new V1Volume()
+                      .name(pvName)
+                      .persistentVolumeClaim(new V1PersistentVolumeClaimVolumeSource()
+                          .claimName(pvcName)))
+                  .addVolumeMountsItem(new V1VolumeMount()
+                      .mountPath("/shared")
+                      .name(pvName)))
+              .adminServer(new AdminServer()
+                  .serverStartState("RUNNING")
+                  .adminService(new AdminService()
+                      .addChannelsItem(new Channel()
+                          .channelName("default")
+                          .nodePort(0))
+                      .addChannelsItem(new Channel()
+                          .channelName("T3Channel")
+                          .nodePort(0))))
```

Comment:
This port needs to match with the t3channel port of the domain you are creating. Otherwise it won't work.

Reply:
The test runs fine. I checked ItDomainOnPV.java. It also set the nodePort(0) in Domain object (see line 257).

Reply:
By setting nodePort(0) here, the operator will pick a free nodePort available for admin T3Channel and default channel.

Reply:
I changed the nodePort in ItDomainOnPV.java , its in PR https://github.com/oracle/weblogic-kubernetes-operator/pull/1678

Just for the T3Channel the port has to match with what is used by the server inside the pod.

Please see the above PR.

Reply:
fixed